packages/e2e/support/hooks.ts

```diff
@@ -47,5 +47,5 @@ After(async function(scenario) {
 });
 
 AfterAll(async function() {
-  await TestRunContext.destroyTestUser();
+  await TestRunContext.destroyTestUsers();
```

Comment:
Testusers? Can there be more than one user per testrun?

Reply:
yes. I'm now storing a pool of test users for that reason. When testing the registration flow, I already have the initial test user created before all the tests. Then I create a new user testing the UI. Then, at the end, I want to remove both of them